custom/abdm/tests/test_abha_creation.py

```diff
@@ -0,0 +1,85 @@
+from unittest.mock import patch
+
+from rest_framework import status
+from rest_framework.reverse import reverse
+from rest_framework.test import APITestCase
+
+from custom.abdm.models import ABDMUser
+
+
+class TestABHACreation(APITestCase):
+
+    def setUp(self):
+        self.user, _ = ABDMUser.objects.get_or_create(username="abdm_test", domain="abdm_test")
```

Comment:
we create the user in `setUpClass` instead to avoid creating or looking for it for each test again.
Similarly, we would clean this up in `tearDownClass`.